src/ai/backend/web/config/unified.py

```diff
@@ -0,0 +1,1144 @@
+        default=None,
+        description="""
+        Default environment for new sessions.
+        """,
+        examples=[None, "python:3.9-ubuntu20.04"],
+        validation_alias=AliasChoices("default_environment", "default-environment"),
+        serialization_alias="default-environment",
+    )
+    default_import_environment: Optional[str] = Field(
+        default=None,
+        description="""
+        Default environment for import operations.
+        """,
+        examples=[None, "python:3.9-ubuntu20.04"],
+        validation_alias=AliasChoices("default_import_environment", "default-import-environment"),
+        serialization_alias="default-import-environment",
+    )
+    menu_blocklist: Optional[CommaSeparatedStrList] = Field(
+        default=None,
+        description="""
+        List of blocked menu items.
+        """,
+        examples=[None, ["admin", "billing"]],
+        validation_alias=AliasChoices("menu_blocklist", "menu-blocklist"),
+        serialization_alias="menu-blocklist",
+    )
+    menu_inactivelist: Optional[CommaSeparatedStrList] = Field(
+        default=None,
+        description="""
+        List of inactive menu items.
+        """,
+        examples=[None, ["maintenance", "experimental"]],
+        validation_alias=AliasChoices("menu_inactivelist", "menu-inactivelist"),
+        serialization_alias="menu-inactivelist",
+    )
+    enable_model_folders: bool = Field(
+        default=True,
+        description="""
+        Whether to enable model folders functionality.
+        """,
+        examples=[True, False],
+        validation_alias=AliasChoices("enable_model_folders", "enable-model-folders"),
+        serialization_alias="enable-model-folders",
+    )
+
+
+class APIConfig(BaseModel):
+    domain: str = Field(
+        default="default",
+        description="""
+        API domain name.
+        """,
+        examples=["api.example.com"],
+    )
+    endpoint: str = Field(
+        default="http://127.0.0.1:8080",
+        min_length=1,
+        description="""
+        API endpoint URL.
+        """,
+        examples=["http://127.0.0.1:8080"],
+    )
+    text: str = Field(
+        default="Backend.AI API",
+        description="""
+        API text description.
+        """,
+        examples=["Backend.AI API"],
+    )
+    ssl_verify: bool = Field(
+        default=True,
+        description="""
+        Whether to verify SSL certificates for API calls.
+        """,
+        examples=[True, False],
+        validation_alias=AliasChoices("ssl_verify", "ssl-verify"),
+        serialization_alias="ssl-verify",
+    )
+    auth_token_name: str = Field(
+        default="sToken",
+        description="""
+        Authentication token name.
+        """,
+        examples=["sToken", "authToken"],
+        validation_alias=AliasChoices("auth_token_name", "auth-token-name"),
+        serialization_alias="auth-token-name",
+    )
+
+
+class RedisHelperConfig(BaseModel):
```

Comment:
This seems to be same with the manager's `RedisHelperConfig`.
I think it would be better to define `RedisHelperConfig` in common package.